docs/legacy-dApp-integration.md

```diff
@@ -285,11 +285,10 @@ AlgoSigner.send({
 The dApp may return the following errors in case of users rejecting requests, or errors in the request:
 
 ```
-    NotAuthorized = '[RequestErrors.NotAuthorized] The extension user does not authorize the request.',
-    InvalidTransactionParams = '[RequestErrors.InvalidTransactionParams] Invalid transaction parameters.',
-    UnsupportedAlgod = '[RequestErrors.UnsupportedAlgod] The provided method is not supported.',
-    UnsupportedLedger = '[RequestErrors.UnsupportedLedger] The provided ledger is not supported.',
-    Undefined = '[RequestErrors.Undefined] An undefined error occurred.',
+    NotAuthorized = '[RequestError.NotAuthorized] The extension user does not authorize the request.',
+    UnsupportedAlgod = '[RequestError.UnsupportedAlgod] The provided method is not supported.',
+    UnsupportedLedger = '[RequestError.UnsupportedLedger] The provided ledger is not supported.',
+    Undefined = '[RequestError.Undefined] An undefined error occurred.',
```

Comment:
Should the readme also have UserRejected and InvalidFormat?

Reply:
InvalidFormat is only for V2, adding UserRejected on a new commit